lobby/build.gradle

```diff
@@ -1,6 +1,6 @@
 buildscript {
```

Comment:
Ditto for removing this `buildscript` configuration, but only after the JUnit 5 Gradle plugin dependency is removed, similar to what was done in the `game-core` subproject.